### PR TITLE
Add ww-metadata file with mp android chart info

### PR DESCRIPTION
**Pull request by JotraN**
Add ww-metadata file with mp android chart info.

### CODE REVIEW

**Review by ifeomaro — APPROVED** [review]

**Comment by JotraN** [comment]
Got the A-OK to merge this in - as long as we don't merge this into the actual origin `PhilJay/MPAndroidChart` we should be fine.

**Pull request merged**
